src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -546,7 +546,6 @@ export default {
 			max-width: 100%;
 			white-space: nowrap;
 			text-overflow: ellipsis;
-			padding-left: 6px;
```

Comment:
I don't have a strong opinion about this padding-left here, but if you remove it here, you also have to remove it here:
https://github.com/nextcloud/nextcloud-vue/blob/063cb379aae4cf13ab949d1411fad9cf884cb242/src/components/AppNavigationSettings/AppNavigationSettings.vue#L123-L129

Otherwise the settings button label won't be aligned with the appnavigationitem labels anymore.
Sorry for being so picky 🙈

Reply:
oh no please dont say sorry, this is right, not picky. Thats why we have reviewers